ajax/libs/simpleCookie/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "simplecookie",
```

```suggestion
  "name": "simpleCookie",
```
I think name should match the directory casing for consistency.